api_v3/services/PartnerService.php

```diff
@@ -76,13 +76,12 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 			$partner_registration = new myPartnerRegistration ( $parentPartnerId );
 			
 			$ignorePassword = false;
-			if ($existingUser && ($this->getKs()->partner_id == Partner::ADMIN_CONSOLE_PARTNER_ID || $this->getKs()->partner_id == $parentPartnerId)){
```

Comment:
$parentPartnerId = $this->getKs()->partner_id;
so there is no need to compare that to itself